src/game/server/ddracecommands.cpp

```diff
@@ -419,7 +423,8 @@ void CGameContext::ConMuteIP(IConsole::IResult *pResult, void *pUserData)
 		pSelf->Console()->Print(IConsole::OUTPUT_LEVEL_STANDARD, "mutes",
 				"Invalid network address to mute");
 	}
-	pSelf->Mute(&Addr, clamp(pResult->GetInteger(1), 1, 86400), NULL);
+	const char *pReason = pResult->NumArguments()>2 ? pResult->GetString(2) : "No reason given";
```

Comment:
If there is no reasom given it shouldnt show the `(No reason given)` at all, just the mute message.